dietpi/dietpi-software

```diff
@@ -1396,8 +1396,12 @@ Available commands:
 		aSOFTWARE_CATX[$software_id]=15
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/vpn/#tailscale'
 		aSOFTWARE_INTERACTIVE[$software_id]=1
+		# Disable support for all distro
```

Comment:
that's not needed at all as there is no other distro we support /being available. On Stretch we don't do any updates anymore. Means it can be enabled for distro.